api-contract/src/test/java/uk/gov/hmcts/payment/api/contract/FeeDtoTest.java

```diff
@@ -28,7 +30,9 @@ public void negativeFeeVolumeShouldFailValidation() {
 
         Set<ConstraintViolation<FeeDto>> violations = validator.validate(feeDto);
         Iterator<ConstraintViolation<FeeDto>> iterator = violations.iterator();
-        assertThat(iterator).extracting(ConstraintViolation::getMessage)
+        assertThat(StreamSupport.stream(violations.spliterator(), false)
+            .collect(Collectors.toList()))
+            .extracting(ConstraintViolation::getMessage)
```

Comment:
You could simplify this with:
`assertThat(violations.stream().toList()).extracting(...)`

The same comment applies to changes below